.ci_support/linux_64_mpimpichscalarcomplex.yaml

```diff
@@ -42,11 +42,11 @@ pin_run_as_build:
   suitesparse:
     max_pin: x.x
 ptscotch:
-- 6.0.9
+- 7.0.2
 scalar:
 - complex
 scotch:
-- 6.0.9
+- 7.0.2
 suitesparse:
 - '5'
 superlu_dist:
```

Comment:
The variant of `superlu_dist 7.1.1` that's mpi-enabled [requires](https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fsuperlu_dist-7.1.1-h25dcc4a_1.tar.bz2) `mpich >=3.4.2,<4.0a0`, which is incompatible with the `mpich 4` pin here. You need to update `superlu_dist` to 7.2.0 (or we need to rebuild superlu_dist 7.1.1 for newer mpich).

Reply:
Well, that person could be you? 🙃 

I don't know if petsc relies on superlu_dist in any meaningful way. If you can remove it without affecting most use-cases, then sure 🤷

Reply:
> Well, that person could be you? 🙃

I cannot commit even more time in things I do not use. The SuperLU support in PETSc is one of such things

> I don't know if petsc relies on superlu_dist in any meaningful way. 

Think about superlu_dist as an optional plugin that provides extra features to PETSc.

> If you can remove it without affecting most use-cases, then sure 🤷

That's the case. For sure some users may be affected, but it is precisely those users the ones who should commit to maintain the superlu_dist feedstock.